### PR TITLE
Change HCP variable in SIGNALCONSTANTS

### PR DESCRIPTION
**Pull request by azpeitrn**
SFT-FIRSTTEST-SignalsLogin

### CODE REVIEW

File: src/test/java/com/pki/spotfire/qa/at/signals/ui/SIGNALSCONSTANTS.java [hunk 1]
```diff
@@ -11,5 +11,5 @@
     public static final String SIGNALS_USER = (System.getProperty("signals.user") != null) ? System.getProperty("signals.user") : "tmdemo";
     public static final String SIGNALS_PASSWORD = (System.getProperty("signals.password") != null) ? System.getProperty("signals.password") : "P4ssworD";
 
-    public static final String HCP = "Signals";
+    public static final String SIGNALS = "Signals";
```
Comment (oskarmora) [inline src/test/java/com/pki/spotfire/qa/at/signals/ui/SIGNALSCONSTANTS.java]: not agree with this naming convention, I propose blba bla bla

Comment (azpeitrn) [inline src/test/java/com/pki/spotfire/qa/at/signals/ui/SIGNALSCONSTANTS.java]: Could you please be more clear in your comments? :)

Comment (oskarmora) [inline src/test/java/com/pki/spotfire/qa/at/signals/ui/SIGNALSCONSTANTS.java]: I was saying that bal bla bl a

Comment (azpeitrn) [inline src/test/java/com/pki/spotfire/qa/at/signals/ui/SIGNALSCONSTANTS.java]: Ahhh, I understand

**Pull request merged**